client/wildcard/src/components/Tooltip/Tooltip.story.tsx

```diff
@@ -53,43 +61,77 @@ Basic.parameters = {
     },
 }
 
+export const DisabledTrigger: Story = () => (
+    <Tooltip content="Tooltip still works properly">
+        <Button variant="primary" disabled={true} style={{ pointerEvents: 'none' }}>
+            Disabled Button 🚫
+        </Button>
+    </Tooltip>
+)
+
+DisabledTrigger.parameters = {
+    chromatic: {
+        disable: true,
+    },
+}
+
+export const Pinned: Story = () => (
+    <>
+        <Tooltip content="My tooltip" defaultOpen={true}>
+            Example
+        </Tooltip>
+
+        <p>
+            <small>
+                (A pinned tooltip is shown when the target element is rendered, without any user interaction needed.)
+            </small>
+        </p>
+    </>
+)
+
+Pinned.parameters = {
+    chromatic: {
+        disable: true,
```

Comment:
```suggestion
        enableDarkMode: true,
        disableSnapshot: false,
```

Given this one is pinned, it'd be cool if we could visually snapshot this. Not essential if we run into issues, but would be useful to avoid future regressions!

Reply:
I agree! Seems like the original Tooltip had to wait on a delay before showing the pinned Tooltip, but ours should render open initially without any issue. :)